### DISCUSSION TITLE
Getting Error - TemplateNotFound(template)

### DISCUSSION THREAD

**Discussion by askillinda24** [pallets/flask#5365] [Q&A] [answered]
Getting below error though I placed the templates in right place and right directory.

"jinja2.exceptions.TemplateNotFound: auth/register.html"

I followed the instructions until the end of below page and tried to run my flask app
https://flask.palletsprojects.com/en/3.0.x/tutorial/templates/

Got the above error.

Expected to see register page without any errors
http://127.0.0.1:5000/auth/register

Environment:

- Python version: 3.11
- Flask version: 3.0.0

**Comment by ThiefMaster** [comment]
> I placed the templates in right place and right directory.

My guess would be that you didn't. Paste a directory tree (e.g. output from the `tree` command).

**Reply by askillinda24** [reply]
thanks for your reply, here you go for my directory structure:

![image](https://github.com/pallets/flask/assets/151886566/f25fdb12-d8bb-40b8-a118-34457dcf1727)

**Reply by askillinda24** [reply]
Below are template HTMLs under templates folder:

![image](https://github.com/pallets/flask/assets/151886566/6235ad20-e334-4d5b-ad2c-fa9e5a78968b)

**Reply by ThiefMaster** [reply]
ugh that output on windows is useless. `tree /f` should be better. and please post it as plaintext (in a code block) and not as an image.

**Reply by ThiefMaster** [reply, marked as answer]
Read the error again. It mentions `auth/register.html` - where's your `auth` subdirectory? :)

**Reply by askillinda24** [reply]
ah, I got it now
very silly mistake, i just blindly followed tutorial :(

**Reply by askillinda24** [reply]
now it works, but seems below part is not working.

{% extends 'base.html' %}

I think i need to give absolute path of the base.html

**Reply by askillinda24** [reply]
thanks @ThiefMaster
